packages/modules/electricity_tariffs/energycharts/tariff.py

```diff
@@ -0,0 +1,39 @@
+from typing import Dict
+from datetime import datetime, timedelta
+import json
+import urllib.request
+
+from modules.common.abstract_device import DeviceDescriptor
+from modules.common.component_state import TariffState
+from modules.common.configurable_tariff import ConfigurableElectricityTariff
+from modules.electricity_tariffs.energycharts.config import EnergyChartsTariffConfiguration
+from modules.electricity_tariffs.energycharts.config import EnergyChartsTariff
+
+
+def fetch_prices(config: EnergyChartsTariffConfiguration) -> Dict[int, float]:
+    current_dateTime = datetime.now()
+    tomorrow = datetime.now() + timedelta(1)
+    start_time = current_dateTime.strftime("%Y-%m-%d") + 'T00%3A00%2B01%3A00'
+    end_time = tomorrow.strftime("%Y-%m-%d") + 'T23%3A59%2B01%3A00'
+    url = f'https://api.energy-charts.info/price?bzn={config.country}&start={start_time}&end={end_time}'
+    add_price = config.serve_price
+    a = urllib.request.urlopen(url)
+    raw_prices = json.loads(a.read().decode())
```

Comment:
```suggestion
    raw_prices = req.get_http_session().get(url).json()
```
Hier bitte das Requests-Modul verwenden. Wenn Du die Session aus packages/modules/common/req.py verwendest, wird im Callback-Hook der Fehlerstatus geprüft und die Antwort automatisch geloggt. Dann muss das nicht bei jeder Abfrage implementiert werden.